frontend/ui/widget/filechooser.lua

```diff
@@ -332,6 +332,9 @@ function FileChooser:genItemTableFromPath(path)
                 file_item.bold = not file_item.bold
             end
         end
+        if self.clipboard and self.clipboard[full_path] then -- selected file
+            file_item.dim = true
+        end
```

Comment:
Could it be better, in select mode, to have everything dimmed and only what is selected non-dimmed ?